src/llm_vm/onsite_llm.py

```diff
@@ -122,7 +122,7 @@ def load_finetune(self, model_filename):
         self.model.load_state_dict(torch.load(os.path.join(model_path_default,"finetuned_models", self.model_name, model_filename)))
 
 
-    def generate(self,prompt,max_length=100,**kwargs): # both tokenizer and model take kwargs :(
+    def generate(self,prompt,max_length=100, *tokenizer_kwargs, **generation_kwargs): # both tokenizer and model take kwargs :(
```

Comment:
Should it be` **tokenizer_` ?

Reply:
Double star

Reply:
Why single vs double

Reply:
yeah forgot * is for non keyword args. changing this now